client/lib/reader-lists-feeds/constants.js

```diff
@@ -0,0 +1,6 @@
+module.exports.action = {
```

Comment:
Is `export const action = { RECEIVE_READER_LIST_FEEDS = 'RECEIVE_READER_LIST_FEEDS', ... }` acceptable? There seems to be a mixture of the two approaches in the Calypso codebase.